### DISCUSSION TITLE
如何让面积图从最左侧开始绘制

### DISCUSSION THREAD

**Discussion by WLyKan** [klinecharts/KLineChart#604] [Q&A]
现在绘图都是从中间偏右开始绘制的，而且一天的走势图还会超出一屏。如何设置从最左边开始绘制，并且一天的走势在一屏内画完

**Comment by sidgwick** [comment]
请问你解决了吗? 我也想知道这个需求怎么实现.
这里还有个难点是, 11:30 到 01:00 的休市时间, 还需要从图上剔除出去
